src/core_plugins/markdown_vis/public/markdown_vis_params.html

```diff
@@ -1,20 +1,43 @@
 <div class="kuiSideBarSection">
   <div class="markdown-vis-options form-group">
     <div>
-      <label for="markdownVisInput">Markdown</label>
-      <small class="pull-right"><a target="_window" href="https://help.github.com/articles/github-flavored-markdown/">Help <i aria-hidden="true" class="fa fa-link"></i></a></small>
+      <label
+        for="markdownVisInput"
+        i18n-id="markdownVis.params.markdownLabel"
+        i18n-default-message="Markdown"
+      >
+      </label>
+      <small class="pull-right">
+        <a target="_window" href="https://help.github.com/articles/github-flavored-markdown/">
+          <span
+            i18n-id="markdownVis.params.helpLinkLabel"
+            i18n-default-message="Help"
+          >
+          </span>
+          <i aria-hidden="true" class="fa fa-link"></i>
+        </a>
+      </small>
     </div>
     <div class="kuiSideBarFormRow">
       <label class="kuiSideBarFormRow__label" for="markdownVisFontSize">
-        Font Size (<span ng-bind="editorState.params.fontSize"></span>pt)
+        <span
+          i18n-id="markdownVis.params.fontSizeLabel"
+          i18n-default-message="Font Size ({fontSize}pt)"
+          i18n-values="{ fontSize: editorState.params.fontSize }"
+        >
+        </span>
       </label>
       <div class="kuiSideBarFormRow__control">
         <input id="markdownVisFontSize" type="range" ng-model="editorState.params.fontSize" class="form-control" min="8" max="36" />
       </div>
     </div>
     <div class="kuiSideBarFormRow">
-      <label class="kuiSideBarFormRow__label" for="markdownVisOpenLinksInNewTab">
-        Open links in new tab
+      <label
+        class="kuiSideBarFormRow__label"
+        for="markdownVisOpenLinksInNewTab"
+        i18n-id="markdownVis.params.openLinksLabel"
+        i18n-default-message="Open links in new tabdown"
```

Comment:
Somehow my initial comment vanished here, sorry... typo: `tabdown` -> `tab`

Reply:
Sorry, I've just fixed it. Thank you for the comment.